docs/guides/on-demand.mdx

```diff
@@ -251,18 +251,15 @@ The upload locally method includes a two-step process:
 
 ```js
 const uploadAsset = async () => {
-  const response = await axios.post(
-    "https://livepeer.studio/api/asset/request-upload",
-    {
-      headers: {
-        Authorization: `Bearer ${process.env.LP_API_KEY}`,
-        "Content-Type": "application/json",
-      },
-      data: JSON.stringify({
-        name: "Example name",
-      }),
-    }
-  );
+  const response = await axios.post("https://livepeer.studio/api/asset/request-upload", {
+    headers: {
+      Authorization: `Bearer ${process.env.LP_API_KEY}`,
+      "Content-Type": "application/json",
+    },
+    data: JSON.stringify({
+      name: "Example name",
+    }),
+  });
   console.log(JSON.stringify(response.data));
```

Comment:
Maybe this should also return the response, as a developer will likely want that in their code.
```suggestion
  console.log(JSON.stringify(response.data));
  return response.data;
```